Sources/Concordium/Domain/Types.swift

```diff
@@ -478,7 +484,7 @@ public struct EntrypointName: Serialize, Deserialize, Equatable {
     }
 
     public static func deserialize(_ data: inout Cursor) -> Self? {
-        guard let parsed = data.readString(withLengthPrefix: UInt16.self) else { return nil }
+        guard let parsed = data.readString(lengthPrefix: UInt16.self) else { return nil }
```

Comment:
why not `prefixLength` its like mean `length of the prefix`, instead of `prefix of the length`  😶

Reply:
Sure, might make more sense